install.sh

```diff
@@ -73,7 +82,7 @@ helm install md-helm-operator fluxcd/helm-operator \
 ### APPLY THE TEMPLATE OUTPUTS
 
 # first deploy kubedb because its needed for harbor to install properly
-kubectl apply -f tmp/moondog/templates/kubedb.yaml
+applyall tmp/moondog/templates/kubedb/*.yaml
```

Comment:
I see no way that adding `--wait` to this could cause issues, just pointing out in case somebody else who knows more attaches significance to it

Reply:
Good catch. I didn't do it on purpose. But yeah I think it doesn't really matter.

Reply:
i dont think it does either